ZuluContent/Zulu/Engines/Loot/LootGenerator.cs

```diff
@@ -177,15 +192,32 @@ protected override void OnTarget(Mobile from, object targeted)
         public static void MakeLoot(Container container, LootTable table)
         {
             var lootItems = table.Roll();
+            var itemsDict = new Dictionary<string, Item>();
 
-            foreach (var item in lootItems)
+            foreach (var lootItem in lootItems)
             {
-                if (!(item.Is<BaseWeapon>() || item.Is<BaseShield>() || item.Is<BaseClothing>() || item.Is<BaseJewel>() ||
-                      item.Is<BaseArmor>() || item.Is<BaseTool>() || item.Type == typeof(Pickaxe)) || MakeItemMagical(item))
+                if (!(lootItem.Is<BaseWeapon>() || lootItem.Is<BaseShield>() || lootItem.Is<BaseClothing>() || lootItem.Is<BaseJewel>() ||
+                      lootItem.Is<BaseArmor>() || lootItem.Is<BaseTool>() || lootItem.Type == typeof(Pickaxe)) || MakeItemMagical(lootItem))
                 {
-                    item.CreateIn(container);
+                    Item item = lootItem.Create();
+                    if (item.Stackable)
+                    {
+                        if (itemsDict.TryGetValue(item.GetType().ToString(), out var dictLi))
+                            dictLi.Amount++;
```

Comment:
This should be `dictLi.Amount += item.Amount` no? otherwise you're not merging stacks together, just increasing the initial amount by one.

Reply:
I think it just generated seperate items unstacked so it wouldn't matter but this is safer